src/Controls/src/Core/Layout/AbsoluteLayout.cs

```diff
@@ -50,7 +62,21 @@ public static Rect GetLayoutBounds(BindableObject bindable)
 			return (Rect)bindable.GetValue(LayoutBoundsProperty);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/AbsoluteLayout.xml" path="//Member[@MemberName='SetLayoutFlags'][1]/Docs/*" />
+		/// <summary> Sets the layout flags of a view that will be used to interpret the layout bounds set on it when it is added to the layout.</summary>
+		/// <remarks>
+		/// <para>This method supports the <c>AbsoluteLayout.LayoutFlags</c> XAML attached property. In XAML, Application developers can specify the following <see cref = "T:Microsoft.Maui.Layouts.AbsoluteLayoutFlags" /> enumeration value names for the value of this property on the children of a <see cref = "T:Microsoft.Maui.Controls.AbsoluteLayout" />:</para>
```

Comment:
You should be able to actually reference types like `<c>AbsoluteLayout.LayoutFlags</c>` with `<see cref="AbsoluteLayout.LayoutFlags" />` that will add a nice link to the right type in the online docs and a clickable link in the Visual Studio tooltip :)